build/repo.props

```diff
@@ -1,5 +1,5 @@
 <Project>
   <ItemGroup>
-    <ExcludeFromTest Include="$(RepositoryRoot)\test\Microsoft.VisualStudio.LanguageServices.Razor.Test\Microsoft.VisualStudio.LanguageServices.Razor.Test.csproj" Condition="'$(OS)'!='Windows_NT'" />
```

Comment:
Having the '\' after `$(RepositoryRoot)` doesn't work in this instance, I'll change it to just removing the '\' (I missed the conditionality).